drivers/storage/portworx/component/portworx_basic.go

```diff
@@ -262,8 +262,8 @@ func (c *portworxBasic) createClusterRole() error {
 				},
 				{
 					APIGroups: []string{"storage.k8s.io"},
-					Resources: []string{"storageclasses"},
-					Verbs:     []string{"get", "list"},
+					Resources: []string{"storageclasses", "csinodes", "volumeattachments"},
+					Verbs:     []string{"get", "list", "create", "delete", "update"},
```

Comment:
this is giving more privileges to storageclasses than needed. See comment: https://github.com/portworx/px-installer/pull/958#discussion_r595588078